### ISSUE TITLE
Flaky Test: HiveOprationSuite - TProtocolException: Required field 'sessionHandle' is unset

### ISSUE THREAD

**Issue by turboFei** [apache/incubator-kyuubi#2521]
### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### Search before asking

- [X] I have searched in the [issues](https://github.com/apache/incubator-kyuubi/issues?q=is%3Aissue) and found no similar issues.

### Describe the bug

```
HiveOperationSuite:
- get catalogs !!! CANCELED !!!
  org.apache.commons.lang3.SystemUtils.isJavaVersionAtMost(JAVA_1_8) was false (HiveEngineTests.scala:36)
- get schemas !!! CANCELED !!!
  org.apache.commons.lang3.SystemUtils.isJavaVersionAtMost(JAVA_1_8) was false (HiveEngineTests.scala:44)
- get tables !!! CANCELED !!!
  org.apache.commons.lang3.SystemUtils.isJavaVersionAtMost(JAVA_1_8) was false (HiveEngineTests.scala:67)
- get columns !!! CANCELED !!!
  org.apache.commons.lang3.SystemUtils.isJavaVersionAtMost(JAVA_1_8) was false (HiveEngineTests.scala:135)
- get functions !!! CANCELED !!!
  org.apache.commons.lang3.SystemUtils.isJavaVersionAtMost(JAVA_1_8) was false (HiveEngineTests.scala:210)
- get table types !!! CANCELED !!!
  org.apache.commons.lang3.SystemUtils.isJavaVersionAtMost(JAVA_1_8) was false (HiveEngineTests.scala:229)
- get primary keys !!! CANCELED !!!
  org.apache.commons.lang3.SystemUtils.isJavaVersionAtMost(JAVA_1_8) was false (HiveEngineTests.scala:244)
- get cross reference !!! CANCELED !!!
  org.apache.commons.lang3.SystemUtils.isJavaVersionAtMost(JAVA_1_8) was false (HiveEngineTests.scala:269)
- basic execute statements, create, insert query !!! CANCELED !!!
  org.apache.commons.lang3.SystemUtils.isJavaVersionAtMost(JAVA_1_8) was false (HiveEngineTests.scala:316)
- test getTypeInfo !!! CANCELED !!!
  org.apache.commons.lang3.SystemUtils.isJavaVersionAtMost(JAVA_1_8) was false (HiveEngineTests.scala:334)
- test setClientInfo !!! CANCELED !!!
  org.apache.commons.lang3.SystemUtils.isJavaVersionAtMost(JAVA_1_8) was false (HiveEngineTests.scala:432)
- test get query id !!! CANCELED !!!
  org.apache.commons.lang3.SystemUtils.isJavaVersionAtMost(JAVA_1_8) was false (HiveOperationSuite.scala:44)
- test set command *** FAILED ***
  org.apache.thrift.protocol.TProtocolException: Required field 'sessionHandle' is unset! Struct:TExecuteStatementReq(sessionHandle:null, statement:set hive.query.name=test, runAsync:true)
  at org.apache.hive.service.rpc.thrift.TExecuteStatementReq.validate(TExecuteStatementReq.java:629)
  at org.apache.hive.service.rpc.thrift.TCLIService$ExecuteStatement_args.validate(TCLIService.java:5591)
  at org.apache.hive.service.rpc.thrift.TCLIService$ExecuteStatement_args$ExecuteStatement_argsStandardScheme.write(TCLIService.java:5648)
  at org.apache.hive.service.rpc.thrift.TCLIService$ExecuteStatement_args$ExecuteStatement_argsStandardScheme.write(TCLIService.java:5617)
  at org.apache.hive.service.rpc.thrift.TCLIService$ExecuteStatement_args.write(TCLIService.java:5568)
  at org.apache.thrift.TServiceClient.sendBase(TServiceClient.java:71)
  at org.apache.thrift.TServiceClient.sendBase(TServiceClient.java:62)
  at org.apache.hive.service.rpc.thrift.TCLIService$Client.send_ExecuteStatement(TCLIService.java:239)
  at org.apache.hive.service.rpc.thrift.TCLIService$Client.ExecuteStatement(TCLIService.java:231)
  at org.apache.kyuubi.engine.hive.operation.HiveOperationSuite.$anonfun$new$4(HiveOperationSuite.scala:60)
  ...
Run completed in 4 seconds, 861 milliseconds.
Total number of tests run: 1
Suites: completed 3, aborted 0
Tests: succeeded 0, failed 1, canceled 15, ignored 0, pending 0
*** 1 TEST FAILED ***
Error:  Failed to execute goal org.scalatest:scalatest-maven-plugin:2.0.2:test (test) on project kyuubi-hive-sql-engine_2.12: There are test failures -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal org.scalatest:scalatest-maven-plugin:2.0.2:test (test) on project kyuubi-hive-sql-engine_2.12: There are test failures
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute (MojoExecutor.java:306)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:211)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:165)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:157)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:121)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:127)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:294)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:960)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:293)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:196)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:566)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
Caused by: org.apache.maven.plugin.MojoFailureException: There are test failures
    at org.scalatest.tools.maven.TestMojo.execute (TestMojo.java:109)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute (MojoExecutor.java:301)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:211)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:165)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:157)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:121)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:127)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:294)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:960)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:293)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:196)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:566)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java)
```

### Affects Version(s)

1.6.0

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

**Comment by wForget** [comment]
fixed in #2519

**Comment by turboFei** [comment]
Closed by #2519
